ext/ddtrace.h

```diff
@@ -1,5 +1,6 @@
 #ifndef DDTRACE_H
 #define DDTRACE_H
+#include "components-rs/common.h"
```

Comment:
Minor nit, but code style: we're using <> includes for everything which is not a path relative to the current file.